src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int totalSupply = 0;
+        int totalBuy = 0;
+        int result;
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                String[] parts = line.split(",");
+                String operation = parts[0];
+                int amount = Integer.parseInt(parts[1]);
+
+                if (operation.equals("supply")) {
+                    totalSupply += amount;
+                } else if (operation.equals("buy")) {
+                    totalBuy += amount;
+                }
+            }
+        } catch (IOException exception) {
+            throw new RuntimeException("Can't read file.");
+        }
+
+        result = totalSupply - totalBuy;
 
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName))) {
+            writer.write("supply," + totalSupply);
+            writer.newLine();
+            writer.write("buy," + totalBuy);
+            writer.newLine();
+            writer.write("result," + result);
+            writer.newLine();
+        } catch (IOException exception) {
+            throw new RuntimeException("Can't write to file.");
+        }
```

Comment:
enormous big method that violates SRP, use this approach instead
```suggestion
        String dataFromFile = readFile(fromFileName);
        String report = createReport(dataFromFile);
        writeReportToFile(report, toFileName);
```